content/runbook/server_monitor.md

```diff
@@ -60,14 +60,6 @@ rapidly filling up:
 
 -   **PostgreSQL** PostgreSQL is the data store for the Chef Infra
     Server.
--   **RabbitMQ** The RabbitMQ data folder can fill up if the
```

Comment:
@PrajaktaPurohit Do we need content on how to monitor Elasticsearch?

Reply:
Might be good to add as a next iteration - I will make note.